flake.nix

```diff
@@ -38,6 +38,7 @@
             buildInputs = oldAttrs.buildInputs
               ++ [ pkgs.nixpkgs-fmt ]
               ++ (with pkgs.haskellPackages; [
+              cabal-install
```

Comment:
Interesting, I must've only used local cabal when building this repository

Reply:
I don't have local cabal 😎